src/utils/getDiscourseRelations.ts

```diff
@@ -1,47 +1,50 @@
-import type { InputTextNode, TextNode } from "roamjs-components/types/native";
+import type {
```

Comment:
`getDiscourseRelations` should just be a refactor.  It was done some time ago and note sure why I left it in this PR